lib/management/configuration.js

```diff
@@ -171,6 +171,21 @@ function patchConfiguration(instanceId, newConf, log, cb) {
                         };
                     }
                     break;
+                case 'location-scality-sproxyd-v1':
+                    location.type = 'scality';
+                    if (l.details && l.details.bootstrapList &&
+                        l.details.proxyPath) {
+                        locations.details = {
+                            connector: {
+                                sproxyd: {
+                                    chordCos: l.details.chordCos || null,
```

Comment:
We may want to add ARC support here as well, or not (@RachedBenMustapha ?)

Reply:
@jonathan-gramain you're the sproxyd guru so correct me if I'm wrong :) We rely on already-deployed sproxyd and I believe ARC settings will be bound to the `proxyPath` we are clients of.

Reply:
Right, since we're using the `proxyPath` to an existing sproxyd that's what matters. But then is the chord COS used at all, aren't we passing the object bucket/key in the path directly?

Reply:
@jonathan-gramain chord cos is passed to sproxydclient